prdoc/pr_5765.prdoc

```diff
@@ -0,0 +1,31 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: ...
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Added foreign locations to local accounts converter to all the parachains
```

Comment:
You added `HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>`. I'd include that here

Reply:
done